wallet/src/de/schildbach/wallet/ui/send/FeeCategory.java

```diff
@@ -27,18 +27,18 @@
 	/**
 	 * We don't care when it confirms, but it should confirm at some time. Can be days or weeks.
 	 */
-	ECONOMIC(Coin.valueOf(1000)), // 0.02 mBTC
+	ECONOMIC(Coin.valueOf(10000)), // 0.02 mBTC
```

Comment:
This is something we forgot to change, but now that dash fees are at 1000, changes to this file are unnecessary